Checker Error Message

ChrisZYJ opened the pull request:
## Description

Makes case file error messages clearer and more obvious

![image](https://github.com/user-attachments/assets/c905da73-0d4b-4bd7-91b5-b124f19a9e2f)

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Something else

sbryngelson commented:
This was intentional. I don't like the use of long strings of characters because how they appears will depend on the width of your terminal and its font. It seems very old school and somehow bothers me. It also makes it harder to check for such patterns in the future, since I do it via `grep` in CI. I'm open to alternative ways to make the printing of case file errors more readable, but don't want to introduce long patterns of `----` or `=====` or `!!!!!!`. Does that make sense?

ChrisZYJ commented:
Thanks for teaching me good coding practices! I've moved the `---` and reverted my changes to your grep implementation. It now formats the Fortran output messages better and emphasizes them using colors in Mako. Please let me know if this is better.

sbryngelson commented on .github/workflows/lint-source.yml:
```diff
@@ -40,7 +40,7 @@ jobs:
 
     - name: Looking for junk code
       run: |
-        ! grep -iR -e '\.\.\.' -e '\-\-\-' -e '===' ./src/* | grep '!'
```
what's the reason for this change?

sbryngelson replied:
oh i see, you're just reverting the change it sounds like

ChrisZYJ replied:
yes!
